README.rst

```diff
@@ -20,6 +20,10 @@ Features
 **Validate multiple files at once**
   Either with the same schema, or different ones.
 
+**Validate multiple files in parallel with multiprocessing**
+  specify `-p <number_of_processes>` or `--processes <number_of_processes>` to
+  parallelize across up to that many processes. (it defaults to 1 when unspecified)
+
```

Comment:
Ah sorry, I was unclear. Can you move this into it's own section "Command Line Args" after the "Testing" section?